src/lib/storage/encoding_type.hpp

```diff
@@ -82,8 +82,8 @@ std::ostream& operator<<(std::ostream& stream, const SegmentEncodingSpec& spec);
 
 using ChunkEncodingSpec = std::vector<SegmentEncodingSpec>;
 
-inline constexpr std::array all_encoding_types{EncodingType::Unencoded,        EncodingType::Dictionary,
-                                               EncodingType::FrameOfReference, EncodingType::FixedStringDictionary,
-                                               EncodingType::RunLength,        EncodingType::LZ4};
+inline constexpr std::array encoding_types{EncodingType::Unencoded,        EncodingType::Dictionary,
+                                           EncodingType::FrameOfReference, EncodingType::FixedStringDictionary,
+                                           EncodingType::RunLength,        EncodingType::LZ4};
```

Comment:
Is the following applicable?
```suggestion
inline constexpr auto encoding_types = magic_enum::enum_values<EncodingType>();
```

Reply:
We did not touch that location. The diff is caused by merging the master branch into that branch. As you showed in one of your PRs, it is applicable.

Reply:
Shall I change it there or do you want to do so?

Reply:
I'll change it in this PR.